src/Application.php

```diff
@@ -35,15 +35,16 @@ class Application extends BaseApplication
      */
     public function middleware($middleware)
     {
-        // Catch any exceptions in the lower layers,
-        // and make an error page/response
-        $middleware->push(new ErrorHandlerMiddleware());
+        $middleware
+            // Catch any exceptions in the lower layers,
+            // and make an error page/response
+            ->push(new ErrorHandlerMiddleware();
```

Comment:
@ADmad I wondered last time why you added the Application tests and I thought that it was to help users to write their own when adding Middleware.